sdk/core/core/src/az_span.c

```diff
@@ -391,6 +391,45 @@ AZ_NODISCARD az_result az_span_append_int64(az_span * self, int64_t n) {
   return _az_span_builder_append_uint64(self, n);
 }
 
+static AZ_NODISCARD az_result _az_span_builder_append_uint32(az_span * self, uint32_t n) {
+  if (n == 0) {
+    return az_span_append(*self, AZ_SPAN_FROM_STR("0"), self);
+  }
+
+  uint32_t div = 1000000000;
+  uint32_t nn = n;
+  while (nn / div == 0) {
+    div /= 10;
+  }
+
+  while (div > 1) {
+    uint8_t value_to_append = _az_decimal_to_ascii((uint8_t)(nn / div));
+    AZ_RETURN_IF_FAILED(az_span_append(*self, az_span_init(&value_to_append, 1, 1), self));
+
+    nn %= div;
+    div /= 10;
+  }
+
+  uint8_t value_to_append = _az_decimal_to_ascii((uint8_t)nn);
+  return az_span_append(*self, az_span_init(&value_to_append, 1, 1), self);
+}
+
+AZ_NODISCARD az_result az_span_append_uint32(az_span * self, uint32_t n) {
```

Comment:
Match signature with az_span and az_span *

Reply:
done